test/instrumentation/modules/http/request.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+var agent = require('../../_agent')()
+
+var http = require('http')
+
+var test = require('tape')
+var express = require('express')
+var request = require('request')
```

Comment:
`request` isn't a dependency

Reply:
Oops. I guess something else depends on it, so I didn't notice. 😅